packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DocumentTemplate.tsx

```diff
@@ -48,7 +48,12 @@ export default function DocumentTemplate(props: DownloadDocumentTemplateProps) {
         alignItems={'center'}
         justifyContent={'flex-start'}
         gap={2}
-        sx={{ backgroundColor: 'tertiary.100', p: 2.5, my: 2 }}
+        sx={{
+          backgroundColor: 'tertiary.100',
+          p: 2.5,
+          my: 2,
+          flexWrap: { xs: 'nowrap', md: 'wrap-reverse' },
```

Comment:
Per this from Figma: 
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/d4a82af7-1f2e-49bc-85f2-578556f4bd07)